### ISSUE TITLE
模型的训练方法

### ISSUE THREAD

**Issue by wly-ai-bj** [xingyizhou/CenterNet#611]
关于ctdet_coco_dla_1x.sh和ctdet_coco_dla_2x.sh中提到的训练设置，我有几点不太明白。
1x是训练了140个epoch，在90和120个epoch的时候学习率变为原来的十分之一。
2x的训练，你提到两种：
    方法一，是直接训练230个epoch，是在180和210个epoch的时候进行学习率衰减；
    方法二，是在1x训练的基础上进行finetune，我理解这里也是继续原来的训练；
           疑问1:不知这个理解是否准确？
           疑问2: 加载的模型是model_90.pth，并不是1x的最后一个模型model_140.pth，这里是为什么？
           疑问3: 为什么这里没有设置训练的epoch数，如果按照默认值，会是140个epoch，和方法一不 一致；
          疑问4: 学习率衰减的epoch没有设置，意思是用默认的90和120吗？后续的训练学习率保持不变吗？这同样和方法一不一致。

以上是我的几点疑问，麻烦您有时间帮忙答疑解惑。谢谢您！

**Comment by xingyizhou** [comment]
Thank you for your detailed question.
1. Currect.
2. model 90 uses the original learning rate. I haven't tried loading model 140.
3/4. It it 90 + 140 epoch, and drop LR at 90 + 90 and 120 + 90 epoch.

**Comment by sisrfeng** [comment]
Is it necessary to train so many epochs ?
I train hourglass and dla_dcn on the pictures of coco (16k pictures) loading the model trained on coco by the author ,  and calculate AP50 on my own dataset (2.4k pictures). 
Model with heighest AP50 appears at epoch 8, and the heighest  AP50  is 84.
The model at epoch 133 gets 81.5 Ap50.
During the training progress, AP50 is between 76-84
